dim/src/routes/dashboard.rs

```diff
@@ -25,24 +25,18 @@ pub mod filters {
 
     use super::super::global_filters::with_state;
 
-    use tokio::runtime::Handle as TokioHandle;
-
     pub fn dashboard(
         conn: DbConnection,
-        rt: tokio::runtime::Handle,
     ) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
         warp::path!("api" / "v1" / "dashboard")
             .and(warp::get())
             .and(with_auth(conn.clone()))
             .and(with_state::<DbConnection>(conn))
-            .and(with_state::<TokioHandle>(rt))
```

Comment:
Can you manually run cargo fmt pls. Doesn't seem like vscode is picking up the rustfmt file